actions/install-local-artifactory/action.yml

```diff
@@ -2,21 +2,28 @@ name: 'Setup Local Artifactory'
 description: 'Setup Local Artifactory'
 inputs:
   RTLIC:
-    description: 'Local Artifactory License'
+    description: 'Local Artifactory License.'
     required: true
   JFROG_HOME:
-    description: 'JFrog Home'
+    description: 'JFrog Home.'
+    required: false
+  VERSION:
+    description: 'Artifactory Version. Default is latest.'
     required: false
 
 runs:
   using: 'composite'
   steps:
     - name: Install Local Artifactory
       run: |
-        go install github.com/jfrog/jfrog-testing-infra/local-rt-setup@1.2.0
-        # Temporarily use version 7.84.17 due to an issue with the automatic token generation mechanism in 7.90
-        ~/go/bin/local-rt-setup --rt-version 7.84.17
+        go install github.com/jfrog/jfrog-testing-infra/local-rt-setup@main
+        if [ -n "${{ env.VERSION }}" ]; then
+          ~/go/bin/local-rt-setup --rt-version "$VERSION"
+        else
+          ~/go/bin/local-rt-setup
+        fi
```

Comment:
`~/go/bin/local-rt-setup --rt-version ${{ env.VERSION }}`

can't be replaced by 1 line?
@yahavi

Reply:
I'm not sure, @sverdlov93. I don't know how this program will manage an empty flag.